README.md

```diff
@@ -100,6 +101,18 @@ kubectl create namespace greeting-workflow
 kubectl apply -f config/samples/sw.kogito.kie.org__v08_kogitoserverlessworkflow.yaml -n greeting-workflow
 ```
 
+8. Default configuration
+
+A configmap called kogito-builder-config will be created under the kogito-builder namespace at the first application of a workflow, it contains:
```

Comment:
can we add a example here?

Reply:
A screenshot or the yaml ?

Reply:
Or better we provide an install section olm/script